versioned_docs/version-8.5/self-managed/setup/deploy/local/docker-compose.md

```diff
@@ -27,6 +27,10 @@ To start a complete instance of Camunda 8 Self-Managed environment locally:
 1. Download the artifact for Camunda 8 <DockerCompose/>, then extract it.
 2. Enter the extracted directory, and run the following command:
 
+:::note
+This command will only work for enterprise customers that have access to Camunda's private registry
+:::
```

Comment:
```suggestion
For versions earlier than 8.6, the `docker compose up -d` command is only available to enterprise customers with access to Camunda's private registry. However, from version 8.6 onwards, it will work for all users as Web Modeler images are made publicly available.
:::
```